### DISCUSSION TITLE
Branching 0.14

### DISCUSSION THREAD

**Discussion by johnaohara** [Hyperfoil/Horreum#1791] [Announcements]
0.14 has been branched and will be the next stable branch.

the main branch is now 0.15-SNAPSHOT

**Comment by whitingjr** [comment]
I don't see a `0.14` in the tags [list](https://github.com/Hyperfoil/Horreum/tags). Can this get tagged ?

**Reply by johnaohara** [reply]
A tag will be created when 0.14.0 is released, it has not been released yet
